packages/rum-recorder/test/utils.ts

```diff
@@ -51,3 +51,26 @@ export class MockWorker implements DeflateWorker {
     this.pendingMessages.length = 0
   }
 }
+
+export function waitCalls<F extends jasmine.Func>(
+  spy: jasmine.Spy<F>,
+  expectedCallsCount: number,
+  callback: (calls: jasmine.Calls<F>) => void
```

Comment:
I have a question: is it possible to make this return a `Promise<jasmine.Calls<F>>` instead of having a callback parameter?

Reply:
Our IE11 support forbids us to use promises, except in parts where IE11 is not supported anyway (like `fetch` proxy/tests)